contracts/clients/README.md

```diff
@@ -194,52 +223,75 @@ const wallet = await BlsWalletWrapper.connect(
   provider,
 );
 const aggregator = new Aggregator("https://arbitrum-goerli.blswallet.org");
-const estimateFee = await aggregator.estimateFee(bundle); // Remember to include no payment with this bundle
+
+// Create a fee estimate bundle
+const estimateFeeBundle = wallet.sign({
+  nonce,
+  actions: [
+    ...actions, // ... add your user actions here (approve, transfer, etc.)
+    {
+      ethValue: 0,
+      contractAddress: tokenContract.address,
+      encodedFunction: tokenContract.interface.encodeFunctionData("approve", [
+        aggregatorUtilitiesContract.address,
+        1,
+      ]),
+    },
+    {
+      ethValue: 0,
+      contractAddress: aggregatorUtilitiesContract.address,
+      encodedFunction: aggregatorUtilitiesContract.interface.encodeFunctionData(
+        "sendTokenToTxOrigin",
+        [tokenContract.address, 1],
+      ),
+    },
+  ],
+});
+
+const feeEstimate = await aggregator.estimateFee(estimateFeeBundle);
+
+// Add a safety premium to the fee to account for fluctuations in gas estimation
+const safetyDivisor = 5;
+const feeRequired = BigNumber.from(feeEstimate.feeRequired);
+const safetyPremium = feeRequired.div(safetyDivisor);
+const safeFee = feeRequired.add(safetyPremium);
```

Comment:
We aren't using the feeEstimate/safeFee anywhere below.  Does this have anything to do with the bundle example created below?

Reply:
Good catch, updated